alpha: 0
beta: 0
gamma: 10
n: 2
optimization_strategy: 1
